templates/cc/BUILD.bazel

```diff
@@ -29,6 +29,7 @@ go_library(
         "//templates/shared:go_default_library",
         "//vendor/github.com/lyft/protoc-gen-star:go_default_library",
         "//vendor/github.com/lyft/protoc-gen-star/lang/go:go_default_library",
+        "//validate:go_default_library",
```

Comment:
I believe this should already be a transitive dependency here, since it's a dep of `//templates/shared:go_default_library` https://github.com/envoyproxy/protoc-gen-validate/blob/master/templates/shared/BUILD.bazel#L16